<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Frame</class>
 <widget class="QFrame" name="Frame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <widget class="QLineEdit" name="search_tab">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>30</y>
     <width>295</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
	border: none;
	border-radius: 5px;
	background-color: #9067C6;
</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>SEARCH</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame_3">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>30</y>
     <width>350</width>
     <height>433</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #F7ECE1;
border: none;
border-radius: 10px;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="program_title">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>New Program</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="program_code">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>295</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
	border: none;
	border-radius: 5px;
	background-color: #9067C6;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Program Code*</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="program_name">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>186</y>
      <width>295</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
	border: none;
	border-radius: 5px;
	background-color: #9067C6;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Program Name*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="create_button">
    <property name="geometry">
     <rect>
      <x>43</x>
      <y>350</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: none;
	border-radius: 5px;
	background-color: #9067C6;
}

QPushButton::hover {
	background-color: #431480;
	color: white;
}</string>
    </property>
    <property name="text">
     <string>CREATE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancel_button">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>350</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: none;
	border-radius: 5px;
	background-color: #F4D35E;
}

QPushButton::hover {
	background-color: #C1A74C;
	color: white;
}</string>
    </property>
    <property name="text">
     <string>CANCEL</string>
    </property>
   </widget>
   <widget class="QPushButton" name="save_button">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>350</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: none;
	border-radius: 5px;
	background-color: #9067C6;
}

QPushButton::hover {
	background-color: #431480;
	color: white;
}</string>
    </property>
    <property name="text">
     <string>SAVE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="edit_button">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>350</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: none;
	border-radius: 5px;
	background-color: #9067C6;
}

QPushButton::hover {
	background-color: #431480;
	color: white;
}</string>
    </property>
    <property name="text">
     <string>EDIT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delete_button">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>350</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: none;
	border-radius: 5px;
	background-color: red;
	color: white;
}

QPushButton::hover {
	background-color: #C1A74C;
	color: white;
}</string>
    </property>
    <property name="text">
     <string>DELETE</string>
    </property>
   </widget>
   <zorder>save_button</zorder>
   <zorder>edit_button</zorder>
   <zorder>create_button</zorder>
   <zorder>program_title</zorder>
   <zorder>program_code</zorder>
   <zorder>program_name</zorder>
   <zorder>cancel_button</zorder>
   <zorder>delete_button</zorder>
  </widget>
  <widget class="QTableWidget" name="program_table">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>108</y>
     <width>592</width>
     <height>351</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;
border-radius: 10px;
background-color: #CAC4CE;</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustToContents</enum>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="dragDropOverwriteMode">
    <bool>false</bool>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="horizontalScrollMode">
    <enum>QAbstractItemView::ScrollPerItem</enum>
   </property>
   <property name="gridStyle">
    <enum>Qt::NoPen</enum>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <property name="cornerButtonEnabled">
    <bool>false</bool>
   </property>
   <property name="rowCount">
    <number>0</number>
   </property>
   <property name="columnCount">
    <number>2</number>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>44</number>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>295</number>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>PROGRAM CODE</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>PROGRAM NAME</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="no_programs">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>260</y>
     <width>241</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>No Programs Found. Try Adding One!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="new_button">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>30</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border: none;
	border-radius: 5px;
	background-color: #9067C6;
}

QPushButton::hover {
	background-color: #431480;
	color: white;
}</string>
   </property>
   <property name="text">
    <string>NEW</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>CODE</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>80</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>PROGRAM NAME</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
